site/docs/.vuepress/styles/index.scss

```diff
@@ -6,9 +6,52 @@
 }
 
 html.dark {
+  --c-bg-lighter: #2b313a;
   --c-brand-light: #138cb3;
-  --c-tip: #1583a7;
   --c-brand: #009dca;
+  --c-tip: #1583a7;
+  --c-tip-title: var(--c-text-accent);
+  --c-warning: #e0ad15;
+  --c-warning-bg: rgba(255, 197, 23, .05);
+  --c-warning-title: #fdca31;
+  --c-warning-text: rgba(252, 252, 124, 0.9);
+  --c-danger: #fc1e38;
+  --c-danger-bg: rgba(237, 60, 80, .06);
+  --c-danger-title: #fc2d3b;
+  --c-danger-text: rgb(253, 128, 136);
+}
+
+html.dark .custom-container.warning code{
+  color: rgba(252, 210, 83, 0.9);
+  background-color: rgba(255, 197, 23, .1);
+}
+
+html.dark .custom-container.danger code{
+  color: rgb(252, 83, 83);
+  background-color: rgba(237, 60, 80, .15);
+}
+
+html.dark .custom-container.warning a{
+  color: var(--c-warning);
+}
+
+html.dark .custom-container.danger a{
+  color: rgb(253, 54, 54);
+}
+
+html.dark .custom-container.warning blockquote{
+  color: rgba(252, 252, 124, 0.65);
+}
+
+html.dark .custom-container.danger blockquote{
+  color: rgb(253, 128, 136, 0.7);
+}
+
+.custom-container.tip pre code, .custom-container.tip pre[class*="language-"] code,
+.custom-container.warning pre code, .custom-container.warning pre[class*="language-"] code,
+.custom-container.danger pre code, .custom-container.danger pre[class*="language-"] code {
+  color: #fff;
+  background-color: transparent;
```

Comment:
These changes seem to copy the default styles. As far as I can tell, they are not needed, if you write other styles in a correct way